src/Aquarius.Client/Samples/Client/ServiceModel.cs

```diff
@@ -1949,12 +2108,318 @@ public class DeleteUser : IReturnVoid
         public string Id { get; set; }
     }
 
+    [Route("/v2/observationimports", "POST")]
+    public class PostObservationimportV2 : IReturnVoid
```

Comment:
I wonder what the right approach to V2 vs V1 operation names should be.

### Add V2 suffixes to new operations

That's what the current PR approach is doing.

But as the API evolves, there will be a mix of non "Vxxx" suffixes (meaning that operation has not yet been versioned) and Vxx suffixes in the client code. That seems a bit code-smelly to me.

### Add V1 suffixes to old operations

Or you could remove the "V2" suffix from these new operations and only add "V1" to operations which have been versioned. This approach might more strongly promote client code to use the newest versions when possible, but at the right of compile-time errors when you take a new SDK. Maybe that is OK, since the coder would then be prompted to update to the new prefered way, with the option of whacking on a "V1" in a few places just to keep something running.

I have a slight bias towards adding the V1 for replaced methods, but I'm sure there are arguments for either approach. What do you think?